src/ActionModal/components/ActionModal.vue

```diff
@@ -18,6 +18,16 @@
           <i class="material-icons notranslate">close</i>
         </div>
         <span class="action-modal-title">{{ title }}</span>
+        <TmFormMsg
+          v-if="isRestrictedAddressRole"
+          :msg="
+            `Currently you are logged in with a ${session.addressRole} account.
```

Comment:
No! We allow bonding with a "stash" account and nominating with a "controller" account

Reply:
I think it is easier like this. That looks like a rabbit hole to me

Reply:
The idea is that people create their Lunie account like I did and pass funds to it. And then we can go on with our much more user-friendly approach of tying up bonding and nominating together.

Or what are you proposing? That is not a rabbit hole

Reply:
Please follow my proposed plan

Reply:
xD

Reply:
I love it when you are in discussing mood ^^